android/src/test/java/com/revenuecat/purchases/common/PeriodTests.kt

```diff
@@ -2,23 +2,24 @@ package com.revenuecat.purchases.common
 
 import com.revenuecat.purchases.common.mappers.PurchasesPeriod
 import org.assertj.core.api.Assertions.assertThat
-import org.spekframework.spek2.Spek
-import org.spekframework.spek2.style.specification.describe
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
 
-object PeriodTests : Spek({
-    describe("when parsing an invalid PurchasesPeriod") {
-        beforeEachTest {
-            mockLogError()
-        }
-        it("there is no exception") {
-            val period = PurchasesPeriod.parse("365")
-            assertThat(period).isNull()
-        }
+internal class PeriodTests {
```

Comment:
FFTI: Should we name these `PurchasesPeriodTests` so they match the class they test more closely?